datasets/esa-worldcover.yaml

```diff
@@ -29,25 +30,48 @@ Resources:
     Region: eu-central-1
     Type: S3 Bucket
     RequesterPays: False
+    Explore:
+    - '[STAC endpoint](https://services.terrascope.be/stac/)'
 DataAtWork:
   Tutorials:
+  - Title: Exploring the dataset (STAC API examples)
+    URL: https://github.com/ESA-WorldCover/esa-worldcover-datasets
+    AuthorName: VITO
+    AuthorURL: https://esa-worldcover.org
   - Title: Global Fire Spread Prediction System
     URL: https://github.com/SatelliteVu/SatelliteVu-AWS-Disaster-Response-Hackathon
     AuthorName: SatelliteVu
     AuthorURL: https://www.satellitevu.com/
   Tools & Applications:
   - Title: WorldCover Viewer
-    URL: https://viewer.esa-worldcover.org/worldcover/ta
-    AuthorName: ESA/Vito
+    URL: https://viewer.esa-worldcover.org/worldcover/
+    AuthorName: VITO
     AuthorURL: https://esa-worldcover.org/en
   - Title: TerraScope Viewer
     URL: https://viewer.terrascope.be/
     AuthorName: TerraScope
     AuthorURL: https://terrascope.be/en/about-us
+  - Title: ESA Viewer 2020
+    URL: https://worldcover2020.esa.int/viewer
+    AuthorName: ESA
+    AuthorURL: https://worldcover2020.esa.int/
+  - Title: WorldCover GEE App
+    URL: https://vitorsveg.users.earthengine.app/view/worldcover
+    AuthorName: VITO
+    AuthorURL: https://esa-worldcover.org  
   Publications:
+  - Title: ESA WorldCover 10 m 2020 v100
+    URL: https://doi.org/10.5281/zenodo.5571936
+    AuthorName: Zanaga, D., Van De Kerchove, R., De Keersmaecker, W., Souverijns, N., Brockmann, C., Quast, R., Wevers, J., Grosu, A., Paccini, A., Vergnaud, S., Cartus, O., Santoro, M., Fritz, S., Georgieva, I., Lesiv, M., Carter, S., Herold, M., Li, Linlin, Tsendbazar, N.E., Ramoino, F., Arino, O., 2021.
+  - Title: ESA WorldCover 10 m 2021 v200
+    URL: https://doi.org/10.5281/zenodo.7254221
```

Comment:
@dzanaga this url results in error "This DOI cannot be found in the DOI System..."

Reply:
It's pubished now.